100Change2017/etl/compose-and-upload

```diff
@@ -38,6 +40,13 @@ Command-line options:
                                   needed by TorqueDataConnect, and can be optionally, manually, put on
                                   the torque wiki.  We don't automatically do that because we want to
                                   overwrite the configuration out there.
+
+  --pare N                        Reduce the nuber of tiems to 1/N rows, for quick
```

Comment:
"number of items" (two typos there -- ISTR that one or both of them may be ancient?)

Obviously, this same typo problem is repeated throughout the commit.  (I realize there's still some code duplication going on here, and am aware that you are well aware of it!  Still, this is a vast improvement over the previous situation.)